announcement/models/announcement.py

```diff
@@ -84,6 +85,16 @@ def _compute_allowed_user_ids(self):
             announcement.allowed_user_ids = announcement.user_group_ids.users
             announcement.allowed_users_count = len(announcement.user_group_ids.users)
 
+    @api.onchange("is_general_announcement")
+    def _onchange_is_general_announcement(self):
+        self.specific_user_ids = False
+        self.user_group_ids = False
```

Comment:
This would be better in the else to the next if

Reply:
thanks